src/resources/elements/primeDesignSystem/pcard/pcard.scss

```diff
@@ -0,0 +1,114 @@
+@import "styles/styles.scss";
```

Comment:
the prime system must not have dependencies from outside of itself (such as styles.scss).  Need to remove this.